src/styles/index.scss

```diff
@@ -2,2 +1,208 @@
+@import 'variables';
+@import 'stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+// Header block
+
+.header {
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-inline: 50px;
+  }
+
+  &__logo-link {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+
+  &__list {
+    padding: 0;
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__list-item .nav__item {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    height: 60px;
+    margin-left: 20px;
+    font-size: 12px;
+    font-weight: 500;
+    color: #060b35;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: all 0.3s;
+  }
```

Comment:
The selector `.nav__list-item .nav__item` can be simplified according to BEM methodology. Since `.nav__item` is already a BEM element, there's no need to specify its parent block `.nav__list-item` as part of the selector. You can define the styles directly under `.nav__item`.